### PR TITLE
Removed redundant 'new' from button labels

### PR DESCRIPTION
**Pull request by Joelkw**
To be consistent with #17454.

### CODE REVIEW

File: client/web/src/enterprise/extensions/registry/RegistryNewExtensionPage.tsx [hunk 1]
```diff
@@ -217,8 +217,8 @@ export const RegistryNewExtensionPage = withAuthenticatedUser(
                                 {this.state.creationOrError === 'loading' ? (
                                     <LoadingSpinner className="icon-inline" />
                                 ) : (
-                                    <AddIcon className="icon-inline" />
-                                )}{' '}
+                                        <AddIcon className="icon-inline" />
```
Comment (Joelkw) [inline client/web/src/enterprise/extensions/registry/RegistryNewExtensionPage.tsx:220]:
```suggestion
                                    <AddIcon className="icon-inline" />
```

File: client/web/src/enterprise/extensions/registry/RegistryNewExtensionPage.tsx [hunk 2]
```diff
@@ -218,7 +218,7 @@ export const RegistryNewExtensionPage = withAuthenticatedUser(
                                     <LoadingSpinner className="icon-inline" />
                                 ) : (
                                     <AddIcon className="icon-inline" />
-                                )}{' '}
+                                    )}{' '}
```
Comment (Joelkw) [inline client/web/src/enterprise/extensions/registry/RegistryNewExtensionPage.tsx:221]:
```suggestion
                                )}{' '}
```

**Comment by Joelkw** [comment]
@tjkandala  fyi, I'm not sure why the build won't pass; I see the below in puppeteer tests which seems unrelated to my modifications? Is this a known issue/was since resolved? 
image

**Comment by tjkandala** [comment]
This was fixed in #17800, so we just have to re-run the Puppeteer tests

**Review by tjkandala — APPROVED** [review]

**Comment by Joelkw** [comment]
@tjkandala Thanks!

**Pull request merged**